### PR TITLE
Added the ability to fallback to Procyon in the case of decompilation…

### PR DESCRIPTION
**Pull request by jsight**
… failures

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/230/<h2>Build result: FAILURE</span></h2>[...truncated 23439 lines...][JENKINS] Archiving /var/lib/jenkins/workspace/windup-core-PR-dependents-check/windup-quickstarts/victi.ms/pom.xml to org.jboss.windup.quickstarts/windup-victims/2.3.0-SNAPSHOT/windup-victims-2.3.0-SNAPSHOT.pomchannel stoppedArchiving artifactsIRC notifier plugin: Sending notification to: #windupSetting status of 5d15500f7d071b8f0e23e7006432169b859afb80 to FAILURE with url http://10.3.8.5/job/windup-core-PR-dependents-check/230/ and message: Merged build finished.FileNotFoundException means that the credentials Jenkins is using is probably wrong. Or that something is really wrong with github.java.io.FileNotFoundException: https://api.github.com/repos/windup/windup/statuses/5d15500f7d071b8f0e23e7006432169b859afb80	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1625)	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:254)	at org.kohsuke.github.Requester.parse(Requester.java:383)	at org.kohsuke.github.Requester._to(Requester.java:185)	at org.kohsuke.github.Requester.to(Requester.java:160)	at org.kohsuke.github.GHRepository.createCommitStatus(GHRepository.java:702)	at org.kohsuke.github.GHRepository.createCommitStatus(GHRepository.java:714)	at org.jenkinsci.plugins.ghprb.GhprbRepository.createCommitStatus(GhprbRepository.java:140)	at org.jenkinsci.plugins.ghprb.GhprbRepository.createCommitStatus(GhprbRepository.java:122)	at org.jenkinsci.plugins.ghprb.GhprbBuilds.onCompleted(GhprbBuilds.java:118)	at org.jenkinsci.plugins.ghprb.GhprbBuildListener.onCompleted(GhprbBuildListener.java:27)	at org.jenkinsci.plugins.ghprb.GhprbBuildListener.onCompleted(GhprbBuildListener.java:12)	at hudson.model.listeners.RunListener.fireCompleted(RunListener.java:199)	at hudson.model.Run.execute(Run.java:1802)	at hudson.maven.MavenModuleSetBuild.run(MavenModuleSetBuild.java:531)	at hudson.model.ResourceController.execute(ResourceController.java:89)	at hudson.model.Executor.run(Executor.java:240)
Test FAILed.

**Comment by jsight** [comment]
retest

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/862/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/231/<h2>Build result: FAILURE</span></h2>[...truncated 23734 lines...][JENKINS] Archiving /var/lib/jenkins/workspace/windup-core-PR-dependents-check/windup-quickstarts/victi.ms/pom.xml to org.jboss.windup.quickstarts/windup-victims/2.3.0-SNAPSHOT/windup-victims-2.3.0-SNAPSHOT.pomchannel stoppedArchiving artifactsIRC notifier plugin: Sending notification to: #windupSetting status of 5d15500f7d071b8f0e23e7006432169b859afb80 to FAILURE with url http://10.3.8.5/job/windup-core-PR-dependents-check/231/ and message: Merged build finished.FileNotFoundException means that the credentials Jenkins is using is probably wrong. Or that something is really wrong with github.java.io.FileNotFoundException: https://api.github.com/repos/windup/windup/statuses/5d15500f7d071b8f0e23e7006432169b859afb80	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1625)	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:254)	at org.kohsuke.github.Requester.parse(Requester.java:383)	at org.kohsuke.github.Requester._to(Requester.java:185)	at org.kohsuke.github.Requester.to(Requester.java:160)	at org.kohsuke.github.GHRepository.createCommitStatus(GHRepository.java:702)	at org.kohsuke.github.GHRepository.createCommitStatus(GHRepository.java:714)	at org.jenkinsci.plugins.ghprb.GhprbRepository.createCommitStatus(GhprbRepository.java:140)	at org.jenkinsci.plugins.ghprb.GhprbRepository.createCommitStatus(GhprbRepository.java:122)	at org.jenkinsci.plugins.ghprb.GhprbBuilds.onCompleted(GhprbBuilds.java:118)	at org.jenkinsci.plugins.ghprb.GhprbBuildListener.onCompleted(GhprbBuildListener.java:27)	at org.jenkinsci.plugins.ghprb.GhprbBuildListener.onCompleted(GhprbBuildListener.java:12)	at hudson.model.listeners.RunListener.fireCompleted(RunListener.java:199)	at hudson.model.Run.execute(Run.java:1802)	at hudson.maven.MavenModuleSetBuild.run(MavenModuleSetBuild.java:531)	at hudson.model.ResourceController.execute(ResourceController.java:89)	at hudson.model.Executor.run(Executor.java:240)
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/863/
Test PASSed.

File: bootstrap/src/main/java/org/jboss/windup/bootstrap/ConsoleProgressMonitor.java [hunk 1]
```diff
@@ -23,6 +27,7 @@ public void beginTask(String name, int totalWork)
 
         String message = String.format("[%d/%d] %s", currentWork, totalWork, name);
         System.out.println(message);
+        LOG.info(message);
```
Comment (lincolnthree) [inline bootstrap/src/main/java/org/jboss/windup/bootstrap/ConsoleProgressMonitor.java]: Good.

**Pull request merged**